src/app/containers/VestedHistory/components/VestedHistoryRow.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import logoSvg from 'assets/images/tokens/sov.svg';
+import { translations } from 'locales/i18n';
+import { LoadableValue } from 'app/components/LoadableValue';
+import { Asset } from 'types/asset';
+import { LinkToExplorer } from 'app/components/LinkToExplorer';
+import { Icon } from 'app/components/Icon';
+import { VestedHistoryFieldsFragment } from 'utils/graphql/rsk/generated';
+import { useDollarValue } from 'app/hooks/useDollarValue';
+import { DisplayDate } from 'app/components/ActiveUserLoanContainer/components/DisplayDate';
+import { getActionName } from '../utils';
+import { AssetValue } from 'app/components/AssetValue';
+import { AssetValueMode } from 'app/components/AssetValue/types';
+
+interface IVestedHistoryRow {
```

Comment:
It looks like here you might be displaying the staking (voluntary staking) history instead of the vesting history. @pietro-maximoff I'll have a look at the code you're using here and will let you know